### PR TITLE
Add notes about speeding up compile time on qt

### PR DESCRIPTION
**Pull request by ProfessorKaos64**

### CODE REVIEW

**Comment by plexappbuilder** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -56,7 +56,11 @@ Once you've unpacked the Qt 5.6.0 beta package:
 * ``make``
 * ``sudo make install``
 
-That should do it for Qt. It's worth noting that, on a Core i7-950 with 24GB of RAM, this took more than three hours to build.
+That should do it for Qt. It's worth noting that, on a Core i7-950 with 24GB of RAM, this took more than three hours to build. It should be noted that you can prefix `./configure` with a [make](http://linux.die.net/man/1/make) flag, where N is the number of parallel builds. This will significantly speed up compile time. It is [suggested](http://www.makelinux.net/books/lkd2/ch02lev1sec3) to use a number 1 to 2x the # of cores you have. For more, also see [this article](http://www.math-linux.com/linux/tip-of-the-day/article/speedup-gnu-make-build-and-compilation-process).
+
+```
+MAKEFLAGS="-jN" ./configure -confirm-license -opensource
```
Comment (TingPing) [inline README.md]: `MAKEFLAGS="-j$(nproc)"`

Comment (ProfessorKaos64) [inline README.md]: Corrected

**Comment by tru** [comment]
Squash and follow the instructions here: https://github.com/plexinc/plex-media-player/wiki/Contributor-License-Agreement

**Pull request closed without merging**